Client/Frontend/Settings/Clearables.swift

```diff
@@ -84,26 +85,14 @@ class CookiesClearable: Clearable {
     
     func clear() -> Success {
         UserDefaults.standard.synchronize()
-        
         let result = Deferred<Maybe<()>>()
         // need event loop to run to autorelease UIWebViews fully
         DispatchQueue.main.asyncAfter(deadline: .now() + 0.1) {
             // Now we wipe the system cookie store (for our app).
-            let storage = HTTPCookieStorage.shared
-            if let cookies = storage.cookies {
-                for cookie in cookies {
-                    storage.deleteCookie(cookie)
-                }
+            WKWebsiteDataStore.default().removeData(ofTypes: [WKWebsiteDataTypeCookies], modifiedSince: Date(timeIntervalSinceReferenceDate: 0)) {
+                UserDefaults.standard.synchronize()
+                result.fill(Maybe<()>(success: ()))
             }
-            UserDefaults.standard.synchronize()
-            
-            // And just to be safe, we also wipe the Cookies directory.
-            do {
-                try deleteLibraryFolderContents("Cookies", validateClearedExceptFor: [])
```

Comment:
`deleteLibraryFolderContents` doesn't seem to be used anymore, 
same with `deleteLibraryFolder`

Reply:
Removed them thanks!